### ISSUE TITLE
Random integration test failure

### ISSUE THREAD

**Issue by rubensworks** [comunica/comunica#947]
#### Issue type:

- :bug: Bug 

____
#### Description:

As of https://github.com/comunica/comunica/commit/9ad64da7bae9dda857e12b7bb0b3af58d5f0f2af, integration tests are failing randomly:

```
info: Run test: https://comunica.github.io/manifest-ldf-tests/next/sparql/sparql-manifest.ttl#directors01
info: Run test: https://comunica.github.io/manifest-ldf-tests/next/sparql/sparql-manifest.ttl#software02
info: Run test: https://comunica.github.io/manifest-ldf-tests/next/sparql/sparql-manifest.ttl#simple03
info: Run test: https://comunica.github.io/manifest-ldf-tests/next/sparql/sparql-manifest.ttl#common04
info: Run test: https://comunica.github.io/manifest-ldf-tests/next/sparql/sparql-manifest.ttl#missing05
info: Run test: https://comunica.github.io/manifest-ldf-tests/next/sparql/sparql-manifest.ttl#schrodinger06
info: Run test: https://comunica.github.io/manifest-ldf-tests/next/sparql/sparql-manifest.ttl#extends07
info: Run test: https://comunica.github.io/manifest-ldf-tests/next/sparql/sparql-manifest.ttl#belgium08
info: Run test: https://comunica.github.io/manifest-ldf-tests/next/sparql/sparql-manifest.ttl#300triples09
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
lerna ERR! yarn run integration stderr:
Failed to fetch https://comunica.github.io/manifest-ldf-tests/next/sparql/software02/92a942df66f1a1a95f0399f65b06da47491ffdfc, for original resource https://fragments.dbpedia.org/2016-04/en?predicate=http%3A%2F%2Fdbpedia.org%2Fontology%2Fdeveloper&page=2.
Failed to fetch https://comunica.github.io/manifest-ldf-tests/next/sparql/missing05/50a82470eb275a2334fc7f80271f11f2d9b3bf81, for original resource https://data.linkeddatafragments.org/ugent-biblio?predicate=http%3A%2F%2Fxmlns.com%2Ffoaf%2F0.1%2Fname&page=2.
Failed to fetch https://comunica.github.io/manifest-ldf-tests/next/sparql/missing05/e1d4d62c9eac9543199a7ace08f8b8f7fc017d0b, for original resource https://data.linkeddatafragments.org/ugent-biblio?predicate=http%3A%2F%2Fpurl.org%2Fdc%2Fterms%2Fcreator&object=https%3A%2F%2Fbiblio.ugent.be%2Fperson%2F801000154550%23person&page=2.
Failed to fetch https://comunica.github.io/manifest-ldf-tests/next/sparql/missing05/a77a692f1941cfbd67cbe22d53c74244cbef3798, for original resource https://data.linkeddatafragments.org/ugent-biblio?predicate=http%3A%2F%2Fpurl.org%2Fdc%2Fterms%2Fcreator&object=https%3A%2F%2Fbiblio.ugent.be%2Fperson%2F801000072304%23person&page=2.
Failed to fetch https://comunica.github.io/manifest-ldf-tests/next/sparql/300triples09/a05f818b98ee859c13f3f060c100b1120b2cc52d, for original resource https://fragments.dbpedia.org/2016-04/en?page=5.
/home/runner/work/comunica/comunica/packages/actor-dereference-http/lib/ActorDereferenceHttpBase.js:81
                const error = new Error(`Could not retrieve ${action.url} (HTTP status ${httpResponse.status}):\n${bodyString}`);
                              ^

Error: Could not retrieve https://fragments.dbpedia.org/2016-04/en?page=5 (HTTP status 500):
Failed to fetch https://comunica.github.io/manifest-ldf-tests/next/sparql/300triples09/a05f818b98ee859c13f3f060c100b1120b2cc52d, for original resource https://fragments.dbpedia.org/2016-04/en?page=5.
    at ActorDereferenceHttp.run (/home/runner/work/comunica/comunica/packages/actor-dereference-http/lib/ActorDereferenceHttpBase.js:81:31)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async ActorDereferenceRdfParse.run (/home/runner/work/comunica/comunica/packages/bus-dereference/lib/ActorDereferenceParse.js:56:29)
    at async MediatedLinkedRdfSourcesAsyncRdfIterator.getSource (/home/runner/work/comunica/comunica/packages/actor-rdf-resolve-quad-pattern-hypermedia/lib/MediatedLinkedRdfSourcesAsyncRdfIterator.js:61:42)
Emitted 'error' event on UnionIterator instance at:
    at SimpleTransformIterator.destinationEmitError (/home/runner/work/comunica/comunica/node_modules/asynciterator/dist/asynciterator.cjs:1081:23)
    at SimpleTransformIterator.emit (node:events:532:35)
    at MediatedLinkedRdfSourcesAsyncRdfIterator.destinationEmitError (/home/runner/work/comunica/comunica/node_modules/asynciterator/dist/asynciterator.cjs:1081:23)
    at MediatedLinkedRdfSourcesAsyncRdfIterator.emit (node:events:520:28)
    at /home/runner/work/comunica/comunica/node_modules/asynciterator/dist/asynciterator.cjs:177:26
    at MediatedLinkedRdfSourcesAsyncRdfIterator._destroy (/home/runner/work/comunica/comunica/node_modules/asynciterator/dist/asynciterator.cjs:892:9)
    at MediatedLinkedRdfSourcesAsyncRdfIterator.destroy (/home/runner/work/comunica/comunica/node_modules/asynciterator/dist/asynciterator.cjs:174:18)
    at /home/runner/work/comunica/comunica/packages/actor-rdf-resolve-quad-pattern-hypermedia/lib/LinkedRdfSourcesAsyncRdfIterator.js:185:46
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
error Command failed with exit code 1.
lerna ERR! yarn run integration exited 1 in '@comunica/query-sparql'
Error: Process completed with exit code 1.
```

**Comment by jeswr** [comment]
I seem to have encountered a similar issue in https://github.com/comunica/comunica/runs/5709394199?check_suite_focus=true

**Comment by jeswr** [comment]
and https://github.com/comunica/comunica/runs/5718516469?check_suite_focus=true

**Comment by rubensworks** [comment]
The fix for #950 may also be a fix for this. Let's wait a while to observe...
